pkg/network/payload/address.go

```diff
@@ -75,7 +79,7 @@ func NewAddressList(n int) *AddressList {
 
 // DecodeBinary implements Serializable interface.
 func (p *AddressList) DecodeBinary(br *io.BinReader) {
-	br.ReadArray(&p.Addrs)
+	br.ReadArray(&p.Addrs, MaxAddrsCount)
```

Comment:
We should also add null length check here, see https://github.com/neo-project/neo/blob/master/src/neo/Network/P2P/Payloads/AddrPayload.cs#L26.